app.js

```diff
@@ -1190,6 +1190,25 @@ define(function(require) {
 
 			monster.ui.tooltips(parent);
 
+			parent.find('#unassigned_select_all').on('click', function() {
+				var $this = $(this);
+				var $selectAllLabel = $('.unassigned_select_all_label');
+				var isActive = $this.hasClass('active');
+				var $checkboxes = parent.find('#unassigned_numbers .unassigned-number input[type="checkbox"]');
```

Comment:
You should use a single `var` keyword to declare several variables ([rule](https://docs.2600hz.com/ui/docs/javascriptStyleGuide/#variables)).

Reply:
seems I get confused with this part of the documentation: https://docs.2600hz.com/ui/docs/javascriptStyleGuide/#jquery

Reply:
Done, thanks!